packages/react-components/review/react-components.api.md

```diff
@@ -460,6 +460,21 @@ export type LocalizationProviderProps = {
     children: React_2.ReactNode;
 };
 
+// Warning: (ae-internal-missing-underscore) The name "LocalVideoCameraCycleButton" should be prefixed with an underscore because the declaration is marked as @internal
+//
+// @internal
+export const LocalVideoCameraCycleButton: (props: LocalVideoCameraCycleButtonProps) => JSX.Element;
```

Comment:
naming nit: `LocalVideoCameraCycleButton` vs `localVideoCameraSwitcherButtonProps`.

I don't remember which one we decided on, but let's stick to one.

Reply:
ah thanks for catching that, I'll review the naming and make sure it aligns!